app/javascript/react/screens/App/Settings/Settings.scss

```diff
@@ -1,3 +1,7 @@
 .migration-settings h2 {
   margin-bottom: 25px;
 }
+
+.conversion-hosts-list .list-view-pf-main-info {
```

Comment:
This smaller padding is fine for now, but as we add more stuff to the list view items we'll probably want to remove or tweak this to make sure the layout matches the mockups and the patternfly standards. I think maybe once we have more in here, the thicker padding will make more sense.